crates/bin/pcli/src/command/tx.rs

```diff
@@ -858,13 +867,72 @@ impl TxCmd {
                     .full_viewing_key
                     .ephemeral_address(OsRng, AddressIndex::from(*source));
 
+                let timeout_height = match timeout_height {
+                    Some(h) => h.clone(),
+                    None => {
+                        // look up the height for the counterparty and add 2 days of block time
+                        // (assuming 10 seconds per block) to it
+
+                        // look up the client state from the channel by looking up channel id -> connection id -> client state
+                        let mut ibc_channel_client =
+                            IbcChannelQueryClient::new(app.pd_channel().await?);
+
+                        let req = QueryChannelRequest {
+                            port_id: "transfer".to_string(),
```

Comment:
Side note: we hardcode `"transfer"` a lot of places, when referring to the IBC port. Should we not constify that value in the IBC crate somewhere, so we can tersely refer to it as the constant value it is?

Reply:
We do this, it's `Portid::transfer()` in ibc-types. i'll update this pr to use that